README-HE.md

```diff
@@ -0,0 +1,192 @@
+<div dir="rtl" style="direction:rtl;text-align:right;">
+<p>
+<!-- Do not translate this -->
+<details>
+<summary>
+<strong> קרא מדריך זה בשפות אחרות </strong>
+</summary>
+    <ul>
+        <li><a href="./README.md"> English </a></li>
+        <li><a href="./README-CN.md"> 中文 </a></li>
+        <li><a href="./README-RU.md"> русский </a></li>
+        <li><a href="./README-RO.md"> Românesc </a></li>
+        <li><a href="./README-IT.md"> Italiano </a></li>
+        <li><a href="./README-ES.md"> Español </a></li>
+        <li><a href="./README-pt-BR.md"> Português (BR) </a></li>
+        <li><a href="./README-DE.md"> Deutsch </a></li>
+        <li><a href="./README-GR.md"> Ελληνικά </a></li>
+        <li><a href="./README-FR.md"> Français </a></li>
+        <li><a href="./README-KO.md"> 한국어 </a></li>
+        <li><a href="./README-JA.md"> 日本語 </a></li>
+    </ul>
+</details>
+<!-- Do not translate this -->
+
+</p>
+</div>
+<div dir="rtl" style="direction:rtl;text-align:right;">
+<p>
+
+# ברוכים הבאים תורמי קוד פתוח מתחילים!
+
+[![Pull Requests Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat)](http://makeapullrequest.com)
+[![first-timers-only Friendly](https://img.shields.io/badge/first--timers--only-friendly-blue.svg)](https://www.firsttimersonly.com/)
+[![Build Status](https://api.travis-ci.org/freeCodeCamp/how-to-contribute-to-open-source.svg?branch=master)](https://travis-ci.org/freeCodeCamp/how-to-contribute-to-open-source)
+
+להלן רשימת מקורות עבור מתכנתים מתחילים המעוניינים לתרום לקוד פתוח
+
+אם תמצאו מקורות נוספים, אנא תרמו על ידי יצירת pull request
+
+אם יש לכם שאלות או תגובות, אנא צרו issue חדש
```

Comment:
I think that in this instance `comments` should be translated to  `הערות`

Reply:
I think you're right, fixed!